wls-wahlvorbereitung-service/src/test/java/de/muenchen/oss/wahllokalsystem/wahlvorbereitungservice/service/briefwahlvorbereitung/BriefwahlvorbereitungModelMapperTest.java

```diff
@@ -0,0 +1,53 @@
+package de.muenchen.oss.wahllokalsystem.wahlvorbereitungservice.service.briefwahlvorbereitung;
+
+import de.muenchen.oss.wahllokalsystem.wahlvorbereitungservice.domain.Briefwahlvorbereitung;
+
+import java.util.Collections;
+
+import lombok.val;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest(classes = { BriefwahlvorbereitungModelMapperImpl.class })
+class BriefwahlvorbereitungModelMapperTest {
+
+    @Autowired
+    private BriefwahlvorbereitungModelMapper unitUnderTest;
+
+    @Nested
+    class ToModel {
+
+        @Test
+        void isMapped() {
+            val wahlbezirkID = "wahlbezirkID";
+            val entityToMap = new Briefwahlvorbereitung(wahlbezirkID, Collections.emptyList());
```

Comment:
Wir sollten sicherstellen dass auch die Elemente in der Collection gemappt wurden.